File: back-end/Fretefy.Test.WebApi/Dtos/RegiaoCidadeDto.cs
using System;

namespace Fretefy.Test.WebApi.Dtos
{
    public class RegiaoCidadeDto
    {
        public Guid Id { get; set; }
        public Guid RegiaoId { get; set; }
        public Guid CidadeId { get; set; }
        public string RegiaoNome { get; set; }
        public string CidadeNome { get; set; }
    }
}

File: front-end/src/app/Dtos/RegiaoCidadeDto.ts
import { CidadeDto } from './CidadeDto';
import { RegiaoDto } from './RegiaoDto';

export interface RegiaoCidadeDto {
  id: string;
  regiaoId: string;
  regiao?: RegiaoDto;
  cidadeId: string;
  cidade?: CidadeDto;
}
